SCClassLibrary/Common/Control/Buffer.sc

```diff
@@ -341,7 +341,7 @@ Buffer {
 						startFrame = 0, leaveOpen = false, completionMessage;
 		// doesn't change my path
 		^["/b_write", bufnum, path,
-				headerFormat, sampleFormat, numFrames, startFrame,
+				headerFormat, sampleFormat, numFrames.asInt, startFrame.asInt,
```

Comment:
I think the conversion on the language side is appropriate. scsynth should not have to worry about conversing with messy untyped languages. I think it should not be responsible for doing conversions. OSC is typed and that is a good thing. (just my 2c. I'm on a static typing binge lately. it really makes life better.)